main.go

```diff
@@ -43,6 +43,10 @@ type buildOptions struct {
 	makeQEMU   bool
 }
 
+var githubRepoToGitLabProject = map[string]string{
```

Comment:
Thoughts:
Maybe we should hard-code them all, while we're at it?
I understand the lure of assuming `Mender` on all of them, but it does mean that all new ones will slip into this group.
Assuming `Mender` is the least privileged, this isn't really an issue though.

I assume you've been through this already, what did you end up with?

Reply:
I see your point, but hard-coding them will force us to update the code here every single time that we create a repo. As per today, we "only" have to update the code if the new repo is part of the Mender base repositories (i.e. needs to trigger the integration pipeline).